flow/texture.h

```diff
@@ -36,6 +36,9 @@ class Texture {
   // Called on GPU thread.
   virtual void MarkNewFrameAvailable() = 0;
 
+  // Called on GPU thread.
+  virtual void OnUnregistered() = 0;
```

Comment:
`OnTextureUnregistered` instead. Since a subclass is going to override this, `OnUnregistered` will read ambiguously in a class that has other non-texture handling responsibilities.